.github/workflows/uat_hotfix_argo.yml

```diff
@@ -0,0 +1,222 @@
+name: UAT Hotfix (ArgoCD)
+env:
+  OPENSHIFT_SERVER: ${{ secrets.OPENSHIFT_SERVER }}
+  # service account: gitaction
+  OPENSHIFT_TOKEN: ${{ secrets.OPENSHIFT_TOKEN }}
+  OPENSHIFT_TOOLS_NAMESPACE: "3cd915-tools"
+  MS_TEAMS_WEBHOOK_BUILD_CHANNEL: ${{ secrets.MS_TEAMS_WEBHOOK_URI_BUILD_CHANNEL }}
+  AUTH__KEYCLOAK__SECRET: ${{ secrets.KEYCLOAK_SECRET_UAT }}
+  AUTH__KEYCLOAK__SERVICEACCOUNT__SECRET: ${{ secrets.KEYCLOAK_SERVICEACCOUNT_SECRET }}
+  ASPNETCORE_ENVIRONMENT: "uat"
+
+  ## variables for scripts under git\openshift\4.0\scripts\oc-*.sh
+  APP_PORT: 8080
+  DESTINATION: "uat"
+  OC_JOB_NAME: "test"
+  GIT_URL: "${{github.server_url}}/${{github.repository}}"
+  GIT_BRANCH: "test"
+  APP_NAME: "pims"
+  PROJ_PREFIX: "3cd915"
+  PROJ_TOOLS: "3cd915-tools"
+  PROJ_DEV: "dev"
+  PROJ_TEST: "test"
+  PROJ_PROD: "prod"
+  TAG_DEV: "dev"
+  TAG_TEST: "test"
+  TAG_PROD: "prod"
+  INSTANCE: "-uat"
+  NAMESPACE_OVERRIDE: "3cd915-test"
+  DEPLOYMENT_NAMESPACE: "3cd915-test"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  ci-cd-start-notification:
+    name: CI-CD Start Notification to Teams Channel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Start notification to Teams Channel
+        uses: dragos-cojocari/ms-teams-notification@v1.0.2
+        with:
+          github-token: ${{ github.token }}
+          ms-teams-webhook-uri: ${{ env.MS_TEAMS_WEBHOOK_BUILD_CHANNEL }}
+          notification-summary: PIMS UAT Hotfix started.
+          notification-color: 17a2b8
+          timezone: America/Los_Angeles
+
+  build-frontend:
+    name: Build frontend
+    needs: ci-cd-start-notification
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v4
+      - name: Login to OpenShift
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ env.OPENSHIFT_TOKEN }}
+          insecure_skip_tls_verify: true
+          namespace: ${{ env.OPENSHIFT_TOOLS_NAMESPACE }}
+      - name: Call script to build frontend (pims-app and pims-app-base)
+        run: |
+          ./openshift/4.0/player.sh build app-base -apply
+          ./openshift/4.0/player.sh build app -apply
```

Comment:
You'll need to build the proxy as well (that is in my PR for the scheduler microservice).

Reply:
The proxy DOES get built in the next step (bundled with the API build). This step is for the frontend. Did I miss something else?